php/helpers.php

```diff
@@ -65,6 +65,31 @@ function block_value( $key ) {
 	return block_field( $key, false );
 }
 
+/**
+ * Convenience method to return the block configuration.
+ *
+ * @return array
+ */
+function block_config() {
+	global $block_lab_config;
+	return $block_lab_config;
+}
+
+/**
+ * Convenience method to return a field's configuration.
+ *
+ * @param string $key The name of the field as created in the UI.
+ *
+ * @return array|null
+ */
+function block_field_config( $key ) {
+	global $block_lab_config;
+	if ( ! isset( $block_lab_config['fields'][ $key ] ) ) {
+		return null;
```

Comment:
@lukecarbis should this return a new WP_Error or an empty array instead?  `null` feels a bit uneasy.

Reply:
@rheinardkorf I chose `null` for the following reasons:

1. `block_field()` also returns `null` on error
2. I don't want to return a `WP_Error`, since I believe it's reasonable to call `block_field_config()` to check whether the config is set or not. e.g. `if ( isset( block_field_config( 'foo' ) )`
3. I don't want to return `false`, since that might be mistaken for an expected value (for example, an unset checkbox). This is more true for `block_field()`, but the same _sort of_ applies here

Reply:
We'd have to document this then. Just to avoid warnings and possible exceptions that can be thrown because of the `null`.  If you're happy with that course of action I'll update my review.